### PR TITLE
Fix refresh to see reservations issues

### PR DESCRIPTION
**Pull request by ibugithub**
In this branch 
I solved getting an unauthorized status when loading the reservations page the first time after logging in without refreshing.

[The issue was when the reservation component was sending requests to the backend It had a token null in the request headers]
[Actually when we dispatch a request from a component to slice It just runs the dispatched element, not the whole slice component, So I had to take the token from local storage inside the dispatched element and now it's working successfully ]

### CODE REVIEW

File: src/redux/reducers/resereveSlice.js [hunk 1]
```diff
@@ -4,12 +4,6 @@ import axios from 'axios';
 const baseUrl = process.env.REACT_APP_API_AUTH_URL;
```
Comment (lincoln1883) [inline src/redux/reducers/resereveSlice.js:4]: As discussed please address this naming issue.
```
REACT_APP_API_URL=http://localhost:3001/api/v1
REACT_APP_API_AUTH_URL=http://localhost:3001
```
so you wont need line (6). eg.  
``` 
const BASE_URL = `${process.env.REACT_APP_API_AUTH_URL}/login
```

Comment (ibugithub) [reply src/redux/reducers/resereveSlice.js:4]: Oh, ok

Comment (lincoln1883) [reply src/redux/reducers/resereveSlice.js]: This woks perfectly.
Screenshot from 2023-10-08 08-35-24

Comment (ibugithub) [reply src/redux/reducers/resereveSlice.js]: Working on it. The issue is We are not getting the updated user from local storage when logging out and logging in again.

**Review by lincoln1883 — APPROVED** [review]
Hi @ibugithub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights

- Fixed lingering reservation bug ✔️
- Fixed logout bug✔️
- Used Git flow ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

**Pull request merged**